lib/avro/builder/dsl.rb

```diff
@@ -58,7 +58,7 @@ def type(name, type_name, options = {}, &block)
       end
 
       # Lookup an Avro schema object by name, possibly fully qualified by namespace.
-      def lookup(key, required: true)
+      def lookup(key)
```

Comment:
Not new but should `type` and `lookup` be private? We don't want people authoring DSLs to use them right?